app/models/permission.rb

```diff
@@ -23,6 +23,7 @@ def self.with_translations
   end
 
   def self.reset_resources
+    BookmarkControllerValidator.reset_controllers_list
```

Comment:
This function is only called once, from the bookmark test. I would prefer resetting the validator in that test rather then here, `Permission` shouldn't really need to know about the internals of this validator.